ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -429,6 +441,18 @@ export const WorkspaceAbout = fp.flow(
                   </div>
                 )}
             </div>
+            {workspace &&
+              WorkspacePermissionsUtil.isOwner(workspace.accessLevel) && (
```

Comment:
Does this link work for any owner? Or does it require direct access to the billing account?

Reply:
It works with any owner.  I guess it comes from this role: https://github.com/broadinstitute/terraform-terra/blob/master/scripts/project-setup-scripts/terra-billing-project-owner.yaml#L9

Reply:
I see, thanks. Historically our preferred UX pattern when turning off some functionality like this is to disable rather than hide. I suggest using tooltip trigger + disabled on the link. Example:

https://github.com/all-of-us/workbench/blob/main/ui/src/app/pages/admin/admin-workspace.tsx#L152-L163

Reply:
Thanks! Do you happen to have an example to do the similar 'navigate' function to an external site?

Reply:
Ah, yeah there's no simple way with `StyledExternalLink` currently. If you're willing - adding a disabled property to this component would make sense, similar to what we have on StyledRouterLink: https://github.com/all-of-us/workbench/blob/main/ui/src/app/components/buttons.tsx#L528-L532

Reply:
Thanks! It works!